asyncio_run_in_process/_child.py

```diff
@@ -204,8 +205,10 @@ def _run_process(parent_pid: int, fd_read: int, fd_write: int) -> None:
                     _do_async_fn(async_fn, args, to_parent, loop),
                 )
             except BaseException:
-                _, exc_value, exc_tb = sys.exc_info()
-                # `mypy` thingks that `exc_value` and `exc_tb` are `Optional[..]` types
+                exc_type, exc_value, exc_tb = sys.exc_info()
+                # `mypy` thinks that `exc_value` and `exc_tb` are `Optional[..]` types
+                if exc_type == asyncio.CancelledError:
```

Comment:
*nitpick*

`==` comparisons like this are fine but I tend to go with `is` comparisons since classes are singletons.